xr_sample_project/Assets/SampleViewer/Samples/VRSample/Scripts/LocationSelector.cs

```diff
@@ -170,6 +214,36 @@ public void GoToChristchurchNewZealand()
         continuousMovement.SetSpeed(50f);
         continuousMovement.SetVerticalSpeed(15f);
     }
-    
+
+    public void GoToNewYork()
```

Comment:
These methods are really bad code. Please refactor to not have any method specific to a location. They all just repeat the same few lines.